### DISCUSSION TITLE
Paladino de Khalmyr hab Enganação

### DISCUSSION THREAD

**Discussion by pereiraad** [YuriAlessandro/gerador-ficha-tormenta20#227] [Perguntas e Respostas] [answered]
Olá, gerei uma ficha e caiu um paladino de Khalmyr  com a origem membro de guilda e veio a pericia Enganação, então, eu não sei se seria um erro, porém não sei se poderia um paladino (ainda mais de Khlamyr) ter essa pericia, digo poder ter até pode (por isso não acho que é um erro) porém não seria nada funcional. Por acaso já teriam visto essa opção e optaram  por deixar, além do mais é umas ficha de nimb ou só ainda não haviam avaliado essa situação?

**Comment by YuriAlessandro** [comment, marked as answer]
Não tínhamos visto a situação ainda. E acho que pelo menos por enquanto vamos manter assim. Eu sempre acho bem interessante pensar em como esse tipo de coisa se encaixa no RP: um Paladino bom de enganação, de blefe, mas q ao mesmo tempo precisa ser justo o tempo todo, como ele lida com isso?

Você ser bom em alguma coisa não significa que você precisa praticar essa coisa, precisa exercer, isso pode ser até uma questão na vida do personagem.

Quanto ao sistema em si mesmo, a gente deixa Nimb decidir, desde que tudo respeite as regras. Não estamos colocando muitas exceções. Então quando temos um Anão clérigo, assistente de laboratório que é devoto de Lin-Wu podemos sempre pensar que isso não faz sentido, ou pensar: 

"Foi pras terras de jade como assistente de algum inventor armeiro pra combater a tormenta na região de Jade, Encontrou fé por lá depois de uma situação de vida ou morte. Voltou pra Arton pra espalhar os ensinamentos que aprendeu por lá."

E existe sempre a opção de você gerar a ficha mas mudar o que não gostou :D
